src/core/components/files.js

```diff
@@ -119,6 +119,7 @@ function prepareFile (self, file, callback) {
 
   waterfall([
     (cb) => self.object.get(file.multihash, cb),
+    (node, cb) => self.pin.add(file.multihash, cb),
```

Comment:
Why does pin happen here?

Reply:
@diasdavid I saw that adding a file should also pin it, I just put it there to get something working so I could debug the tests, but that's probably not the right place to do it... Open to suggestion on this.